src/App.js

```diff
@@ -1,24 +1,8 @@
-import logo from './logo.svg';
-import './App.css';
+import InputCalculator from './components/Calculator';
```

Comment:
- [ ] Nice start on your readme.  However, all the sections from the [Microverse readme template](https://github.com/microverseinc/readme-template) are required, except for "live demo" and "tests" when not required for the project.  Kindly please add all the missing sections on your readme so your app is well documented.